.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  # Enable version updates for Gradle
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: "main"
```

Comment:
Is the default for dependabot to NOT auto-merge PRs with the dependency updates?
Also, will this setup kick off the PR pipeline checks?  Specifically Unit Tests run?

Reply:
The default for dependabot does not auto-merge PRs with updates. This is an example of a PR that dependabot creates: https://github.com/DataBiosphere/duos-ui/pull/1740